pandas/core/indexes/multi.py

```diff
@@ -1010,14 +1010,14 @@ def _to_safe_for_reshape(self):
 
     def to_frame(self, index=True):
         """
-        Create a DataFrame with the columns the levels of the MultiIndex
+        Create a DataFrame with the columns and levels of the MultiIndex.
```

Comment:
This doesn't seem quite right.  Maybe something like "Create a DataFrame with the levels of the MultiIndex as columns"?

Reply:
Sure, done.